src/redux/initialState.js

```diff
@@ -129,7 +132,8 @@ export default {
       Pan: { showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: null },
       AnnotationEdit: { showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: null },
       TextSelect: { showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: null },
-      MarqueeZoomTool: { dataElement: 'marqueeToolButton', title: 'tool.select', label: 'Marquee Zoom', showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: null }
+      MarqueeZoomTool: { dataElement: 'marqueeToolButton', title: 'tool.select', label: 'Marquee Zoom', showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: null },
+      AnnotationCreateEraser: { showColor: 'never', iconColor: null, currentPalette: null, availablePalettes: [], annotationCheck: annotation => annotation instanceof window.Annotations.FreeHandAnnotation }
```

Comment:
If the eraser tool doesn't have any associated annotations then the value of `annotationCheck` is null. If you are making changes in 5.2 branch, this code is in `map.js`. Let me know if you need any helps with this. Thanks!

Reply:
Ok, I'll update